Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.h

```diff
@@ -127,6 +127,8 @@ class NetworkConnectionToWebProcess
     void transferKeptAliveLoad(NetworkResourceLoader&);
     void setOnLineState(bool);
 
+    void deleteWebsiteDataForOrigins(OptionSet<WebsiteDataType>, const Vector<WebCore::SecurityOriginData>&, CompletionHandler<void()>&&);
```

Comment:
Can we pass a `Span` instead of a `const Vector&`?

Reply:
Not easily. Our current IPC encoder/decode for Span wouldn't work for any non POD type. It encodes/decodes everything as a `const uint8_t*`.